examples/models/11-loading-gltf.lisp

```diff
@@ -0,0 +1,31 @@
+(in-package #:cl-user)
+(defpackage claylib/examples/models-11
+  (:use :cl :claylib)
+  (:export :main))
+(in-package #:claylib/examples/models-11)
+
+(defun pathname-from-relative-path (path)
+  "Return a pathname from a path relative to the claylib project root."
+  (asdf:system-relative-pathname :claylib path))
+
+(defparameter *scene*
+  (make-scene ((raylib-ass (make-model-asset (pathname-from-relative-path
+                                         "examples/models/resources/models/gltf/raylib_32x32.glb"))))
+              ((model (make-model raylib-ass 0 0 0))
+               (grid (make-grid 10 1)))))
+
+(defun main ()
+  (with-window (:title "raylib [models] example - loading gltf")
+    (let ((camera (make-camera-3d 10 10 10
+                                  0 0 0
+                                  0 1 0
+                                  :mode +camera-free+)))
+      (with-scenes *scene*
+        (do-game-loop (:livesupport t)
+          (update-camera camera)
+          (with-drawing
+            ;; TODO Why can't I shadow *claylib-background* in the above let instead?
+            ;; This is a redundant call to clear-background.
+            (clear-background :color +skyblue+)
```

Comment:
We're doing this in core example 10, and probably others, without issue. The one thing I notice is that your `let` is outside of `with-scenes` and `do-game-loop` whereas that example has it inside. Off the top of my head I don't know _why_ that should matter, but give it a shot.

Reply:
It appears not to be working in core example 10, 17, or 27 either :thinking:

Reply:
... Crap. Clearly I didn't look closely enough at the actual examples to verify that our background colors were in fact wrong. :man_facepalming: I'll look into this...